src/core/aria.js

```diff
@@ -10,7 +10,6 @@ sub(
   { once: true }
 );
 
-export function run(conf, doc, cb) {
-  doc.body.setAttribute("aria-busy", "true");
-  cb();
+export function run(conf) {
+  document.body.setAttribute("aria-busy", "true");
```

Comment:
Oh, I thought I'd removed this module? This module is no good. Could you delete it?